### ISSUE TITLE
how can i bypass the prompt to collect annonymous data when i just want to check the phonegap version from the windows command-line

### ISSUE THREAD

**Issue by Annuther** [phonegap/phonegap-app-desktop#843]

**Comment by purplecabbage** [comment]
What version of node are you using?
`node -v`
We have not been able to recreate this issue, so it is hard to tell what is happening.
Something you might try is `phonegap -v --no-insight`

**Comment by Annuther** [comment]
@purplecabbage  - that worked ! thats what i needed  - iread somewhere there was a way to answer once so it would remember your answer - this was it cheers. BTW i have node 4.4.3 

c:\>phonegap -v --no-insight

How you use PhoneGap provides us with important data that we can use to make
our products better. Please read our privacy policy for more information on the
data we collect. http://www.adobe.com/privacy.html

Analytics is off.
If you would like to turn analytics on, simply run `phonegap analytics on`

6.4.4

c:\>

**Comment by purplecabbage** [comment]
Can you try running `phonegap analytics off` ? 
I am curious if this will work for you.
Ideally you should not see the prompt text at all, and it should just output the version number.

**Comment by purplecabbage** [comment]
What I would expect ( or hope ) to see would be : 
```
c:>phonegap analytics off
Analytics is off.
If you would like to turn analytics on, simply run `phonegap analytics on`

c:>phonegap -v
6.4.4
```

**Comment by Annuther** [comment]
hi,
Ok, when i ran the **analytics off** command in the node cli it said permission denied  - when i ran it in the windows command line it works as you stated.

output:
c:\>phonegap analytics off
Analytics is off.
If you would like to turn analytics on, simply run `phonegap analytics on`

c:\>phonegap -v
6.4.4

c:\>

---------------
 I think i ran the initial commands in the node CLI too would that be the problem??

**Comment by purplecabbage** [comment]
yeah, it is most likely that, not sure what you mean by node cli though.
Do you mean the git-bash terminal? That is where I run everything in windows ...

**Comment by Annuther** [comment]
Yes i meant node.js command prompt  i think i had that open since i forgot i had weinre running in the win command prompt in the background so my bad!

cheers man ...

cheers
